packages/ui/src/domainModules/agent/components/AgentDropdownSearch/index.js

```diff
@@ -19,7 +24,14 @@ const defaultProps = {
 }
 
 const extractText = item => {
-  return item && item.attributes && item.attributes.fullName
+  const { attributes } = item
+
+  return (
+    attributes &&
+    ((attributes.disambiguatingDescription &&
+      `${attributes.fullName} ( ${attributes.disambiguatingDescription} )`) ||
+      attributes.fullName)
+  )
```

Comment:
maybe this could be written in a more reader-friendly way.

```
const disambiguatingDescription = objectPath.get(item, 'attributes.disambiguatingDescription')
const fullName = objectPath.get(item, 'attributes.fullName')

return disambiguatingDescription ? `${fullName} (${disambiguatingDescription})` : fullName
```

or this. it's maybe more lines, but super easy to read and understand the first time you read it. the other one has three or four parentheses so it's a bit tricky just to see where each parenthesis ends

```
const disambiguatingDescription = objectPath.get(item, 'attributes.disambiguatingDescription')
const fullName = objectPath.get(item, 'attributes.fullName')

if (disambiguatingDescription) {
  return `${fullName} (${disambiguatingDescription})`
}

return fullName
```